docs/guide/robot_sbc/setup_jetson_nano.md

```diff
@@ -66,76 +136,364 @@ pip3 install -U gdown
+* [Step 2b: Free up the serial port](#step-2b-free-up-the-serial-port-optional-only-needed-if-youre-using-the-robohat-mm1)
+* [Step 3b: Setup Python Environment](#step-3b-setup-python-environment)
+* [Step 4b: (Optional) Install PyGame for USB camera](#step-4b-optional-install-pygame-for-usb-camera)
+
+
+#### Step 1b: Flash Operating System
+
+These instructions work for Jetpack 4.6.1. 
+
+* If you have a 4gb Jetson Nano then download Jetpack 4.6.1 from Nvidia
+  here: [jetson-nano-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jeston_nano/jetson-nano-jp461-sd-card-image.zip)
+* If you have a 2gb Jetson Nano the download Jetpack 4.5.1from Nvidia
+  here: [jetson-nano-2gb-jp461-sd-card-image.zip](https://developer.nvidia.com/embedded/l4t/r32_release_v7.1/jp_4.6.1_b110_sd_card/jetson_nano_2gb/jetson-nano-2gb-jp461-sd-card-image.zip)
+
+
+Visit the official [Nvidia Jetson Nano Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-nano-devkit#prepare)
+or [Nvidia Xavier NX Getting Started Guide](https://developer.nvidia.com/embedded/learn/get-started-jetson-xavier-nx-devkit).
+Work through the __Prepare for Setup__, __Writing Image to the microSD Card__,
+and __Setup and First Boot__ instructions, then return here.
+
+Once you're done with the setup, ssh into your vehicle. Use the terminal for
+Ubuntu or Mac. [Putty](https://www.chiark.greenend.org.uk/~sgtatham/putty/latest.html)
+for windows.
+
+
+Remove Libre Office:
 
 ```bash
-cd ~/projects
+sudo apt-get remove --purge libreoffice*
+sudo apt-get clean
+sudo apt-get autoremove
+```
+
+And add a 6GB swap file:
+
+```bash
+git clone https://github.com/JetsonHacksNano/installSwapfile
+cd installSwapfile
+./installSwapfile.sh
+reboot 
+```
+
+#### Step 2b: Free up the serial port (optional. Only needed if you're using the Robohat MM1)
+
+```bash
+sudo usermod -aG dialout <your username>
+sudo systemctl disable nvgetty
+```
+
+
+#### Step 3b: Setup Python Environment
+
+The installation on JP 4.6.X is more tricky, because we need to use a newer
+version of tensorflow than what is packaged by NVidia for this OS version.
+Also, we need a compatible OpenCV version which needs to be built on the nano.
+
+* Step 1: Install mamba-forge
+
+Download and install mambaforge
+
+```bash
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-aarch64.sh
+chmod u+x ./Mambaforge-Linux-aarch64.sh
+bash ./Mambaforge-Linux-aarch64.sh
 ```
 
-* Get the latest donkeycar from Github.
-> Note: There are currently version updates happening on the `main` branch so you might rather want to checkout the latest stable release as explained below.
+* Step 2: Install git lfs
 
 ```bash
+sudo apt-get install git-lfs
+```
+
+* Step 3: Download Donkey Car
+
+Downloading Donkey Car and the Tensorflow wheel for jp46 and tf29
+
+```bash
+mkdir projects
+cd projects
 git clone https://github.com/autorope/donkeycar
```

Comment:
Failed for me three times at 26%, 26%, 22%:
`Receiving objects:  26% (4234/16216), 37.36 MiB | 9.50 MiB/s`

while working without issue on my desktop on the same network.

I installed GitHub CLI and I was able to pull with that.

Probably not worth noting in the documentation, but mentioning in case someone else has the same problem.

Reply:
Hm, ok, this really should only depend on the internet connection. Git should be good enough, you wouldn't need the GitHub CLI. But good that you mention it. If we see others with the same problem, we'll add a comment here to keep trying.